README.md

```diff
@@ -2,25 +2,33 @@
 Automation of Building your own [Atomic-Host](http://www.projectatomic.io/).
 To know about Atomic-Host, please visit [http://www.projectatomic.io](http://www.projectatomic.io/).
 
-## Steps
+# Steps
 
+## Clone Source
 ```
 $ git clone https://github.com/trishnaguha/build-atomic-host.git
 $ cd build-atomic-host
 ```
 
+## Install Ansible
 Please Make sure Ansible is intalled in your system.
 ```
-$ sudo dnf install ansible
+$ sudo dnf install ansible python2-dnf #Install python2-dnf if you are using Fedora Linux Distribution
```

Comment:
I would remove the python-dnf in the statement. but leave it in the comment. Else this would fail on centos... however, have you tested this on CentOS  ?